launch.go

```diff
@@ -213,20 +232,43 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 	}
 
 	metaByte := []byte("")
-	parentID := 0
 	metaFile := "meta.json" // Write to "meta.json" file
-	metaSource := ""
+	metaLog := ""
+
+	parentBuildIDs := convertToArray(build.ParentBuildID)
```

Comment:
just a concern here: if there is no `build.ParentBuildID` then it **should** return `[]`?

Reply:
Yes, I tried running it in the Golang playground and it looks like `convertToArray` should handle `nil` properly

Reply:
I guess that depends on how Golang would unmarshal it if it's `0`

Reply:
> Yes, I tried running it in the Golang playground and it looks like convertToArray should handle nil properly

What if it's `undefined`?